subtests/docker_cli/run_volumes_selinux/run_volumes_selinux.py

```diff
@@ -149,7 +136,8 @@ def touch_and_check(self, cont, volume, filename, context_pre,
                       get_selinux_context(volume))
         self.logdebug("Touching /tmp/test/%s in container"
                       % filename)
-        os.write(cont.stdin, "touch /tmp/test/%s\necho RET: $?\n" % filename)
+        # Don't just update directory entries with touch, add content too
```

Comment:
ahh good catch.  okay, ya, I'll add comment about why:

Creating a file is a directory (file) update: a separate cache for some filesystems.
Writing to a file (that doesn't exist) touches all the caches (fs directory and pagecache).

This guarantees that when you 'sync', the file is there.  Some filesystems ignore the sync for files with only directory entry changes (until some timeout period happens).